client/app/hosting/framework/runtime/edit/hosting-framework-runtime-edit.controller.js

```diff
@@ -0,0 +1,79 @@
+angular.module("App").controller(
+    "controllers.Hosting.Framework.Runtime.edit",
+    class HostingFrameworkRuntimeEditCtrl {
+
+        /**
+         * @constructs HostingFrameworkRuntimeEditCtrl
+         * @param $scope
+         * @param $stateParams
+         * @param translator
+         * @param Alerter
+         * @param HostingFrameworkRuntime
+         */
+        constructor ($scope, $stateParams, translator, Alerter, HostingFrameworkRuntime) {
+            this.$scope = $scope;
+            this.$stateParams = $stateParams;
+            this.translator = translator;
+            this.Alerter = Alerter;
+            this.HostingFrameworkRuntime = HostingFrameworkRuntime;
+        }
+
+        /**
+         * Initialize HostingFrameworkRuntimeEditCtrl
+         */
+        $onInit () {
+            this.isLoading = true;
+
+            const data = angular.copy(this.$scope.currentActionData);
+            this.entryToEdit = data.runtime;
+
+            this.$scope.edit = () => this.edit();
```

Comment:
We shouldn't pass function to $scope in a ES6 class, ```myCtrl.edit()``` would make more sense in the HTML